src/Resources/Locales/en.json

```diff
@@ -673,6 +673,7 @@
                 "connectionStringInformation": "Data history widgets target data in the Azure Data Explorer instance already connected to this Azure Digital Twins instance.",
                 "connectionLoadingText": "Loading connection information...",
                 "noConnectionInformationText": "No connection information found",
+                "connectionStringInvalid": "Invalid connection string!",
```

Comment:
Feel free to suggest any other content here @msnyder-msft @pasanchMSFT

Reply:
yeah. let's remove the ! no need to yell at them. :)
Personally, I'd rather do "Connection string is invalid"

Reply:
I agree, we should be ok without `!` .